testing/endpoint.go

```diff
@@ -323,8 +325,19 @@ func (endpoint *Endpoint) QueryConnectionHandshakeProof() (
 	return clientState, clientProof, consensusProof, consensusHeight, connectionProof, proofHeight
 }
 
+var sequenceNumber int
+
+func (endpoint *Endpoint) IncrementNextChannelSequence() {
+	if endpoint.disableUniqueChannelIDs {
+		return
+	}
+	sequenceNumber++
+	endpoint.Chain.App.GetIBCKeeper().ChannelKeeper.SetNextChannelSequence(endpoint.Chain.GetContext(), uint64(sequenceNumber))
+}
+
```

Comment:
This is the core of the change. We force the increment of the `NextChannelSequence` entry so that every chan has a different ID.